app/src/main/java/com/github/se/wanderpals/ui/screens/trip/notifications/Notification.kt

```diff
@@ -101,190 +102,243 @@ fun Notification(
-            colors =
-                ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary)) {
-              Row(
-                  horizontalArrangement = Arrangement.spacedBy(8.dp, Alignment.CenterHorizontally),
-                  verticalAlignment = Alignment.CenterVertically,
-              ) {
-                Icon(
-                    imageVector = Icons.Default.Add,
-                    contentDescription = Icons.Default.Add.name,
-                    tint = MaterialTheme.colorScheme.onPrimary,
-                    modifier = Modifier.size(20.dp))
-                Text(
-                    text = "Make an announcement",
-                    style =
-                        TextStyle(
-                            fontSize = 18.sp,
-                            fontWeight = FontWeight(500),
-                            color = MaterialTheme.colorScheme.onPrimary),
-                    textAlign = TextAlign.Center,
+                Modifier.background(MaterialTheme.colorScheme.surfaceTint)
+                    .padding(horizontal = 16.dp, vertical = 8.dp)
+                    .fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically) {
+              Text(
+                  modifier =
+                      Modifier.padding(start = 20.dp, top = 12.dp)
+                          .height(35.dp)
+                          .testTag("notificationTitle"),
+                  text = "Notifications",
+                  textAlign = TextAlign.Center,
+                  style =
+                      MaterialTheme.typography.bodyLarge.copy(
+                          fontWeight = FontWeight.Bold, fontSize = 24.sp),
+                  color = MaterialTheme.colorScheme.onPrimary)
+            }
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceEvenly,
+            verticalAlignment = Alignment.CenterVertically) {
+              Box(modifier = Modifier.weight(1f)) {
+                NavigationButton(
+                    text = "Notifications",
+                    isSelected = isNotificationSelected,
+                    onClick = { onSelectOption(true) },
                 )
               }
+              Box(modifier = Modifier.weight(1f)) {
+                NavigationButton(
+                    text = "Announcements",
+                    isSelected = !isNotificationSelected,
+                    onClick = { onSelectOption(false) })
+              }
             }
       }
-    }
-  }
+}
+
+/**
+ * Composable function for displaying a navigation button.
+ *
+ * @param text The text to display on the button.
+ * @param isSelected Whether the button is currently selected.
+ * @param onClick Callback function for when the button is clicked.
+ */
+@Composable
+fun NavigationButton(text: String, isSelected: Boolean, onClick: () -> Unit) {
```

Comment:
Maybe you can use the SecondaryTabRow composable with two Tabs inside : Notification and Announcement, for optimization and modularity of the code